captum/optim/_param/image/images.py

```diff
@@ -1,538 +1,475 @@
+                image.size(0),
+                image.size(1),
+                image.size(2),
+                image.size(3),
+            )
+            for shared_tensor in self.shared_init
+        ]
+        if self.offset is not None:
+            x = self.apply_offset(x)
+        return (image + sum(x)).refine_names("B", "C", "H", "W")
+
+
+class NaturalImage(ImageParameterization):
+    r"""Outputs an optimizable input image.
+
+    By convention, single images are CHW and float32s in [0,1].
+    The underlying parameterization can be decorrelated via a ToRGB transform.
+    When used with the (default) FFT parameterization, this results in a fully
+    uncorrelated image parameterization. :-)
+
+    If a model requires a normalization step, such as normalizing imagenet RGB values,
+    or rescaling to [0,255], it can perform those steps with the provided transforms or
+    inside its computation.
+
+    Arguments:
+        size (Tuple[int, int]): The height and width to use for the nn.Parameter image
+            tensor.
+        channels (int): The number of channels to use when creating the
+            nn.Parameter tensor.
+        batch (int): The number of channels to use when creating the
+            nn.Parameter tensor, or stacking init images.
+        parameterization (ImageParameterization, optional): An image parameterization
+            class.
+        squash_func (Callable[[torch.Tensor], torch.Tensor]], optional): The squash
+            function to use after color recorrelation. A funtion or lambda function.
+        decorrelation_module (nn.Module, optional): A ToRGB instance.
+        decorrelate_init (bool, optional): Whether or not to apply color decorrelation
+            to the init tensor input.
+    """
+
+    def __init__(
+        self,
+        size: Tuple[int, int] = [224, 224],
+        channels: int = 3,
+        batch: int = 1,
+        init: Optional[torch.Tensor] = None,
+        parameterization: ImageParameterization = FFTImage,
+        squash_func: Optional[Callable[[torch.Tensor], torch.Tensor]] = None,
+        decorrelation_module: Optional[nn.Module] = ToRGB(transform="klt"),
+        decorrelate_init: bool = True,
+    ) -> None:
+        super().__init__()
+        self.decorrelate = decorrelation_module
+        if init is not None:
+            assert init.dim() == 3 or init.dim() == 4
+            if decorrelate_init:
+                assert self.decorrelate is not None
+                init = (
+                    init.refine_names("B", "C", "H", "W")
+                    if init.dim() == 4
+                    else init.refine_names("C", "H", "W")
+                )
+                init = self.decorrelate(init, inverse=True).rename(None)
+            if squash_func is None:
+
+                def squash_func(x: torch.Tensor):
+                    return x.clamp(0, 1)
+
+        else:
+            if squash_func is None:
+
+                def squash_func(x: torch.Tensor):
```

Comment:
Couldn't we just make the `squash_func` variable equal to `torch.sigmoid` as it doesn't require any additional variables like `torch.clamp`?

```
squash_func = torch.sigmoid
```